docs/workflows/quick-start-guide.md

```diff
@@ -1,52 +1,134 @@
 
 # Quickstart Guide
 
-## 1. Upload Data (To an Existing Project on ACED IDP)
-This section outlines how to add data to ACED IDP. Many `g3t` commands behave similarily to `git` commands
+## Dependencies
 
-Initialize a new project
-```
-g3t init aced-demo
+Please ensure you have the following dependencies installed:
+
+* [gen3-client](../requirements/gen3-client.md)
+* [gen3-tracker](../requirements/gen3-tracker.md)
+
+## Usage
+
+`g3t  [OPTIONS] COMMAND [ARGS]...`
+
+The following options and environmental variables are synonymous, you may set them as environmental variables or pass them as parameters to the command line.
```

Comment:
I haven't been able to figure out how to successfully pass the profile name on the command line. In my current workflow I set env variables G3T_PROFILE and G3T_PROJECT. Could you add an example of running a g3t command with options, command, and args that includes setting profile?
```
(g3t_venv) (python311) [lancasru@arc-infra-3 NHW]$ gen3-client configure --profile=cbds --cred=/home/groups/CEDAR/lancasru/.gen3/credentials.json --apiendpoint=https://idp.cbds.ohsu.edu
2024/10/09 10:37:20 A new version of gen3-client is available! The latest version is 2024.10.0. You are using version 2024.09
2024/10/09 10:37:20 Please download the latest gen3-client release from https://github.com/uc-cdis/cdis-data-client/releases/latest
2024/10/09 10:37:20 Profile 'cbds' has been configured successfully.
(g3t_venv) (python311) [lancasru@arc-infra-3 NHW]$ g3t ping
WARNING: No --profile specified, found multiple gen3_client profiles: ['cedar', 'cbds']
msg: 'Configuration ERROR: No profile found'

(g3t_venv) (python311) [lancasru@arc-infra-3 NHW]$ export G3T_PROFILE=cbds
(g3t_venv) (python311) [lancasru@arc-infra-3 NHW]$ g3t ping
msg: 'Configuration OK: Connected using profile:cbds'
endpoint: https://idp.cbds.ohsu.edu
username: lancasru@ohsu.edu

(g3t_venv) (python311) [lancasru@arc-infra-3 NHW]$ export G3T_PROFILE=""
(g3t_venv) (python311) [lancasru@arc-infra-3 NHW]$ g3t ping
WARNING: No --profile specified, found multiple gen3_client profiles: ['cedar', 'cbds']
msg: 'Configuration ERROR: No profile found'

(g3t_venv) (python311) [lancasru@arc-infra-3 NHW]$ g3t ping --profile 'cbds'
WARNING: No --profile specified, found multiple gen3_client profiles: ['cedar', 'cbds']
Usage: g3t ping [OPTIONS]
Try 'g3t ping --help' for help.

Error: No such option: --profile
(g3t_venv) (python311) [lancasru@arc-infra-3 NHW]$ g3t ping --profile cbds
WARNING: No --profile specified, found multiple gen3_client profiles: ['cedar', 'cbds']
Usage: g3t ping [OPTIONS]
Try 'g3t ping --help' for help.

Error: No such option: --profile
(g3t_venv) (python311) [lancasru@arc-infra-3 NHW]$ g3t ping cbds
WARNING: No --profile specified, found multiple gen3_client profiles: ['cedar', 'cbds']
Usage: g3t ping [OPTIONS]
Try 'g3t ping --help' for help.

Error: Got unexpected extra argument (cbds)
```

Reply:
The --profile option goes after g3t but before the first command. Ex: `g3t --profile local ping` and `g3t --profile local init ohsu-test`

Reply:
See [adds example](https://github.com/ACED-IDP/aced-idp.github.io/pull/23/commits/ba970c742d750af18d3ca89a106fb31d390905f6)